modules/library/main/src/test/java/org/geotools/styling/visitor/DuplicatingStyleVisitorTest.java

```diff
@@ -304,6 +305,43 @@ public void testRasterSymbolizerWithUOM() throws Exception {
         assertFalse(Utilities.equals(notEq.getUnitOfMeasure(), rasterSymb.getUnitOfMeasure()));
     }
 
+    public void testRasterSymbolizerWithOverlapBehavior() throws Exception {
+        RasterSymbolizer rasterSymb1 = sf.createRasterSymbolizer();
+        rasterSymb1.setOverlapBehavior(OverlapBehavior.AVERAGE);
+        rasterSymb1.accept(visitor);
+        RasterSymbolizer clone1 = (RasterSymbolizer) visitor.getCopy();
+
+        assertEquals(clone1.getOverlapBehavior(), rasterSymb1.getOverlapBehavior());
+
+        // Try literal expression
+        RasterSymbolizer rasterSymbLiteral = sf.createRasterSymbolizer();
+        rasterSymbLiteral.setOverlap(ff.literal(OverlapBehavior.EARLIEST_ON_TOP.toString()));
+        rasterSymbLiteral.accept(visitor);
+        RasterSymbolizer cloneLiteral = (RasterSymbolizer) visitor.getCopy();
+
+        assertEquals(cloneLiteral.getOverlapBehavior(), rasterSymbLiteral.getOverlapBehavior());
+        assertEquals(cloneLiteral.getOverlap(), rasterSymbLiteral.getOverlap());
+
+        // Try with invalid expression string
+        RasterSymbolizer rasterSymb3 = sf.createRasterSymbolizer();
+        try
+        {
```

Comment:
Code formatting - open braces should be on the same line as the try. Same with the catch statements below.